### PR TITLE
AM2R: add options to flip gameplay

### PR DESCRIPTION
**Pull request by Miepee**
grafik

### CODE REVIEW

**Comment by JeffGainsNGames** [comment]
^ assume that was just missed, but agree it should be added

**Comment by Miepee** [comment]
I originally didnt want to mark it as dangerous, since I personally dont really consider it as such, but it is better for consistency that all chaos options are marked as such.

File: randovania/games/am2r/layout/am2r_configuration.py [hunk 1]
```diff
@@ -87,6 +89,9 @@ def unsupported_features(self) -> list[str]:
         if self.artifacts.required_artifacts > self.artifacts.placed_artifacts:
             result.append("The amount of required DNA cannot be higher than the total amount of placed DNA.")
 
+        if self.horizontally_flip_gameplay or self.vertically_flip_gameplay:
+            result.append("Gameplay is flipped horizontally or vertically.")
```
Comment (henriquegemignani) [inline randovania/games/am2r/layout/am2r_configuration.py:93]: add a test for this

Comment (Miepee) [reply randovania/games/am2r/layout/am2r_configuration.py:93]: Added

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**